example/ooni/dns_injection.cpp

```diff
@@ -35,7 +35,7 @@ int main(int argc, char **argv) {
         exit(1);
     }
 
-    mk::ooni::DnsInjectionTest()
+    mk::ooni::DnsInjection()
```

Comment:
I've merged `DnsInjectionImpl` and `DnsInjectionTest` in a single class called `DnsInjection`. Similar changes below for other OONI tests.